profiles/jerry.profile

```diff
@@ -0,0 +1,2 @@
+CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
```

Comment:
How about putting the profile on the root directory? Actually we won't own other `*.profile` files anyway.

Reply:
Will ShadowNode use the same profile file on every target later? If not, I would prefer put it on profilers directory.

Reply:
So how about profilers/default.profile?

Reply:
Directory profiles already has default.profile and minimal.profile. They are for ShadowNode modules. Perhaps default.jerry.profile is better to show it is profile for jerry.

Reply:
Why not `./profiles/jerry/default.profile`, an implicit separator would make confusion a bit to me :(

Reply:
Well done, that's it!